firstpr/harsha.md

```diff
@@ -1 +1,2 @@
 ##### Harsha Vardhan from india . B.tech student
```

Comment:
Please always use capital `I` for `India`, just like you have done it for your first and last name.